### ISSUE TITLE
Like button / Accessibility: Frame missing title attribute

### ISSUE THREAD

**Issue by JoshuaGoode** [Automattic/wp-calypso#53429]
On both Simple sites and Jetpack/Atomic sites, the Like button renders in a frame which is missing a Title attribute.

Users that check their sites using WCAG accessibility test tools will receive a notice WCAG notice like the following.

**Frames must have title attribute**
"Ensures <iframe> and <frame> elements contain a non-empty title attribute. Applicable success criteria: 2.4.1, 4.1.2"

The iframes do lack a title attribute. 

### Steps to reproduce the behavior

1. Enable or visit a site where Like buttons are enabled. Simple or Atomic/Jetpack.
2. Run the post/page through a WCAG tester such as https://accessibleweb.com/website-accessibility-checker
3. See WCAG issue for lack of Frame title attributes.

### Context

User report, 29972972-hc, 4044316-zen

#### Is this specific to the applied theme? Which one?
No

#### Does this happen on simple or atomic sites or both?
Both

#### Level of impact (Does it block purchases? Does it affect more than just one site?)
Sites that use Like buttons.

#### Reproducibility (Consistent, Intermittent) Leave empty for consistent.
Consistent

**Comment by inaikem** [comment]
Requested a confidence check here before actioning this request: p1623041552252500-slack-C7YPUHBB2.

**Comment by inaikem** [comment]
Raised in turn here: p1623041552252500-slack-C7YPUHBB2

**Comment by JoshuaGoode** [comment]
Noting that this probably belongs over in the Jetpack repo. I'll check and move over as necessary.
